.github/workflows/weekly-checks.yml

```
@@ -66,9 +66,23 @@ jobs:
           GITHUB_TOKEN: ${{ github.token }}
         run: npm run check:external-links -- 'docs/**/*.{md,mdx,ipynb}'
 
+    new-pill-checker:
+      runs-on: ubuntu-latest
+      if: github.repository_owner == 'Qiskit'
+      steps:
+        - uses: actions/checkout@v4
+        - name: Set up Node.js
+          uses: actions/setup-node@v4
+          with:
+            node-version: 18
+        - name: Install Node.js dependencies
+          run: npm ci
+        - name: Check outdated new pills
+          run: npm run check-new-pills'
```

There's a typo that will cause an error. The other suggestion is to use the convention we normally use for checks.

```suggestion
          run: npm run check:new-pills
```